src/main/java/org/broadinstitute/hellbender/transformers/BaseQualityClipReadTransformer.java

```diff
@@ -29,8 +30,8 @@ public BaseQualityClipReadTransformer(int trim_thresh) {
      *
      */
     @Override
-    public GATKRead apply(GATKRead read) {
-        GATKRead readClippedRightEnd = clipReadRightEnd(read);
+    public GATKRead apply(final GATKRead read) {
+        final GATKRead readClippedRightEnd = clipReadRightEnd(read);
         return clipReadLeftEnd(readClippedRightEnd);
     }
 
```

Comment:
I know this exists before this PR, but while you are at it, could you add a documentation to `getRightClipPoint(byte[])` and `getLeftClipPoint(byte[])` explicitly saying that a negative return value means no clipping necessary?